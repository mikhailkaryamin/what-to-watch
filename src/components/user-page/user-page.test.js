import * as React from 'react';
import renderer from 'react-test-renderer';

import UserPage from './user-page.tsx';

describe(`user page`, () => {
  const MockComponent = () => <div />;

  const wrapper = renderer
    .create(
        <UserPage>
          <MockComponent />
        </UserPage>
    ).toJSON();

  test(`should render user page`, () => {
    expect(wrapper).toMatchSnapshot();
  });
});
