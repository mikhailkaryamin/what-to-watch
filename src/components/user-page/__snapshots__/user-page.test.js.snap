// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`user page should render user page 1`] = `
<div
  className="user-page"
>
  <div />
</div>
`;
